subgraphs/balancer-v2/src/mappings/masterchef.ts

```diff
@@ -74,9 +81,11 @@ export function handleLogUpdatePool(event: LogUpdatePool): void {
     return;
   }
   let pool = LiquidityPool.load(poolAddress.value.toHexString());
-  if(!pool){
-    if(FBEETS.toHexString() == poolAddress.value.toHexString()){
-      pool = createLiquidityPool(event,poolAddress.value.toHexString(),"FreshBeets","FBEETS",[FBEETS.toHexString()] )
+  if (!pool) {
+    if (FBEETS.toHexString() == poolAddress.value.toHexString()) {
```

Comment:
what's `FBEETS`?

Reply:
[fBEETS)](https://ftmscan.com/token/0xfcef8a994209d6916eb2c86cdd2afd60aa6f54b1)
FBEETS is a token that can be stacked 
but not create by the vault

Reply:
I see